docs/docs/plugins/02-dev-plugins.md

```diff
@@ -42,31 +42,31 @@ which implies to implement a predefined interface:
 // An ignite plugin must implements the Plugin interface.
 type Interface interface {
 	// Manifest declares the plugin's Command(s) and Hook(s).
-	Manifest() (Manifest, error)
+	Manifest(context.Context) (*Manifest, error)
```

Comment:
wdyt about creating our context like the cosmos-sdk and passing some basic info?

Reply:
It can be something we can explore, specially if you have an initial idea of what we could add to the custom context 👍
Honestly I'm not sure right now what could be useful inside the context.

Users will be able to get information using the bidirectional communication that is implemented in #3544 and #3561. The idea there is that the methods defined in this interface will receive an extra argument that will expose some capabilities to get more specific info. Maybe having more info in the context won't be needed when those are finished.

Reply:
I think context is good for cancelation but not passing data, because it becomes very ambiguous.